ext/drx/drx.h

```diff
@@ -132,8 +132,9 @@ drx_aflags_are_dead(instr_t *where);
 
 /** Flags for \p drx_insert_counter_update */
 enum {
-    DRX_COUNTER_64BIT = 0x01, /**< 64-bit counter is used for update. */
-    DRX_COUNTER_LOCK = 0x10,  /**< Counter update is atomic. */
+    DRX_COUNTER_64BIT = 0x01,    /**< 64-bit counter is used for update. */
+    DRX_COUNTER_LOCK = 0x10,     /**< Counter update is atomic. */
+    DRX_COUNTER_REL_ACQ = 0x100, /**< Release-acquire semantics for counter update. */
```

Comment:
Please add the new feature to the feature changelist at https://github.com/DynamoRIO/dynamorio/blob/master/api/docs/release.dox#L192

Reply:
Done.